examples/resources/akp_cluster/auto_agent_size.tf

```diff
@@ -0,0 +1,50 @@
+data "akp_instance" "example" {
```

Comment:
let's modify the template to include this and custom agent size example in the generated docs. We can just add them after here: https://github.com/akuity/terraform-provider-akp/blob/main/templates/resources/cluster.md.tmpl#L14-L20